main.go

```diff
@@ -224,6 +234,8 @@ func runCLIApp(c *cli.Context) (err error) {
 	}
 
 	logger.Infof("Start gcsfuse/%s for app %q using mount point: %s\n", getVersion(), flags.AppName, mountPoint)
+	logger.Infof("Loading config for mount flags %s", stringify(*flags))
```

Comment:
nit: instead we can write better like, 1GCSFuse mount command flags: %s`

Reply:
Updated logs with above text